compiler/rustc_driver/src/lib.rs

```diff
@@ -743,6 +743,18 @@ fn print_crate_info(
                     }
                 }
             }
+            DeploymentTarget => {
+                use rustc_target::spec::current_apple_deployment_target;
+
+                if sess.target.is_like_osx {
+                    println!("{}", current_apple_deployment_target(&sess.target))
+                } else {
+                    early_error(
+                        ErrorOutputType::default(),
+                        "Only Apple targets currently support deployment version info",
```

Comment:
```suggestion
                        "only Apple targets currently support deployment version info",
```

can you add a `ui` test for this error is emitted

Reply:
Yeah, that works. There's one now that uses the x64 GNU Linux target to test the error branch. The casing is fixed now as well.